libraries/dotnet/EnergyOrigin.IntegrationEvents/README.md

```diff
@@ -1,5 +1,45 @@
-# Library
+# EnergyOrigin.IntegrationEvents
 
-## EnergyOrigin.IntegrationEvents
+## Overview
 
-The purpose of EnergyOrigin.IntegrationEvents is to have a single library for all the Integration Events that are used in the EnergyOrigin solution.
+The EnergyOrigin.IntegrationEvents NuGet package provides a standardized way to define and use integration events,
```

Comment:
*Is our way of sharing integration event contracts between our systems

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1279/commits/ecdb7624ce5509d72f389fd47d16a0b32a0fb93e

Reply:
Fixed for real this time: https://github.com/Energinet-DataHub/energy-origin/pull/1279/commits/c141531366a64b4d76bddf28fd69773917ee3e33